apps/meteor/client/views/room/body/RoomBody.tsx

```diff
@@ -226,17 +208,7 @@ const RoomBody = (): ReactElement => {
 					<div className='messages-container-wrapper'>
 						<div className='messages-container-main' ref={wrapperBoxRefs} {...fileUploadTriggerProps}>
 							<DropTargetOverlay {...fileUploadOverlayProps} />
-							<Box position='absolute' w='full'>
-								{roomLeader ? (
-									<LeaderBar
-										_id={roomLeader._id}
-										username={roomLeader.username}
-										name={roomLeader.name}
-										visible={!hideLeaderHeader}
-										onAvatarClick={handleOpenUserCard}
-										triggerProps={triggerProps}
-									/>
-								) : null}
+							<Box position='absolute' w='full' zIndex={12}>
```

Comment:
I didn't catch why the zIndex here

Reply:
indeed it doesn't seem to make a difference here. i'll remove it :)